tools/ci_build/github/linux/docker/inference/x64/default/cpu/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This file is used by Zip-Nuget Packaging NoContribOps Pipeline,Zip-Nuget-Java Packaging Pipeline
+ARG BASEIMAGE=centos:7
+FROM $BASEIMAGE
+
+ENV PATH /usr/local/gradle/bin:/opt/rh/devtoolset-11/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
why devtoolset-11 vs. devtoolset-10 in aarch64/default/cpu//Dockerfile?

Reply:
aarch64 doesn't have devtoolset 11 yet. I will update it to devtoolset11 when it's ready. 

devtoolset-11 means gcc 11.
devtoolset-10 means gcc 10.